.github/workflows/reusable-release.yaml

```
@@ -27,51 +27,51 @@ jobs:
       contents: read  # Not required for public repositories, but for clarity
     steps:
       - name: Cosign install
-        uses: sigstore/cosign-installer@d58896d6a1865668819e1d91763c7751a165e159
+        uses: sigstore/cosign-installer@d58896d6a1865668819e1d91763c7751a165e159 # v3.9.2
 
       - name: Set up QEMU
-        uses: docker/setup-qemu-action@v3
+        uses: docker/setup-qemu-action@49b3bc8e6bdd4a60e6116a5414239cba5943d3cf # v3.2.0
```

`docker/setup-qemu-action@v3` used version `v3.6.0` in the last workflow run:
https://github.com/aquasecurity/trivy/actions/runs/16640963671/job/47091042143#step:1:26
https://github.com/docker/setup-qemu-action/releases/tag/v3.6.0

```suggestion
        uses: docker/setup-qemu-action@29109295f81e9208d7d86ff1c6c12d2833863392 # v3.6.0
```